src/main/java/mat/server/ConversionMapping.java

```diff
@@ -0,0 +1,24 @@
+package mat.server;
+
+import lombok.*;
```

Comment:
this can be avoided by updating intellij settings, Thanks to @serhii-ilin for suggestion!